lib/repository.py

```diff
@@ -101,7 +101,7 @@ def run(self):
             scripts = []
             print("Downloading scripts...")
             for line in version_file:
-                scripts.append(line.strip('\n').split(','))
+                scripts.append(line.decode("utf-8").strip('\n').split(','))
```

Comment:
The string needs to be unicode, but the urlopen function gives bytes.
We could also decode it using `latin-1` if that is the default behaviour needed.